resources/lib/datafunctions.py

```diff
@@ -652,9 +671,59 @@ def checkVisibility ( self, action ):
             if path[ 1 ].endswith( ")" ):
                 path[ 1 ] = path[ 1 ][:-1]
             return NODE.get_visibility( path[ 1 ] )
-            
-        return ""
+
+        # Audio node visibility
+        elif action.startswith( "activatewindow(musiclibrary,musicdb://" ) or action.startswith( "activatewindow(10502,musicdb://" ) or action.startswith( "activatewindow(MusicLibrary,library://music/" ) or action.startswith( "activatewindow(10502,library://music/" ):
+            path = action.split( "," )
```

Comment:
You're absolutely right :) (See what I mean about a second pair of eyes being useful ;)) I'll sort it :)